pages/forums/view.js

```diff
@@ -0,0 +1,131 @@
+// Module imports
+import React from 'react'
+
+
+
+
+
+// Component imports
+import Component from '../../components/Component'
+import ForumThreadCard from '../../components/Forums/ForumThreadCard'
+import Page from '../../components/Page'
+
+
+
+
+
+// Component constants
+const pageTitle = 'Public Forums'
+
+
+
+
+
+class ForumList extends Component {
+  /***************************************************************************\
+    Private Methods
+  \***************************************************************************/
+
+
+
+  /***************************************************************************\
+    Public Methods
+  \***************************************************************************/
+
+  async componentDidMount() {
```

Comment:
Can we flip this around to use `getInitialProps` like the group profile does?

Reply:
ah yeah that would be cool :P